tests/Transport/HttpTransportTest.php

```diff
@@ -62,9 +62,11 @@ public function testSendDoesNotDelayExecutionUntilShutdownWhenConfiguredToNotDoI
         $config = new Options(['dsn' => 'http://public@example.com/sentry/1']);
         $transport = new HttpTransport($config, $httpClient, MessageFactoryDiscovery::find(), false);
 
-        $transport->send(new Event());
+        $eventId = $transport->send(new Event());
 
         $this->assertAttributeEmpty('pendingRequests', $transport);
+        // null is returned if sending request failed
```

Comment:
Do we return anything else in the other case?

Reply:
Yes. Otherwise the event ID is returned. I added this, because of the confusion here: #905 and here: #886

Reply:
shall I remove this drive by change and create a separate PR out of this?